util/src/test/java/io/kubernetes/client/CopyTest.java

```diff
@@ -34,25 +34,28 @@
 
 /** Tests for the Copy helper class */
 public class CopyTest {
-  private String namespace;
-  private String podName;
-  private String[] cmd;
+
+  private String namespace, podName;
```

Comment:
the diff from this file doesnt seem related w/ the point of this pull? let's keep a minimum change i.e. revert the diff